### DISCUSSION TITLE
Synchronized scrolling

### DISCUSSION THREAD

**Discussion by luis-j** [KDE/ghostwriter#662] [Q&A]
Hi!

I discovered ghostwriter recently, and I think it's fantastic! Thank you for this great software, I already recommended it to my colleagues and some of them are already using it.

I would like to comment the only thing I found inconvenient so far: when HTML preview is active, I can't seem to find the way to keep both panes synchronized, unless I edit and modify the text. But if I navigate the text with the cursor, or scroll either pane with its scroll bar, the two panes are not synchronized making it difficult to see where I am.

Is there a way to achieve this that I'm missing? If not, could this be a useful feature to consider for a future release?

Best,

luis

**Comment by Sci-Mon** [comment]
Even cooler: There is no need for scrolling the preview. There is the possibility to show only this much text in the preview as it's showing in the editor. I recall that markdownpad is doin this and I found it quite convenient. So there is **one** scrollbar for the editor and the preview.

**Advantage**:
- not the entire document has to be rendered at once but only part of it.
    - less CPU power and therefore
    - faster behaviour
    - less distraction because of
- More space left without the scrollbar of the preview
- WYSIWYG

**Disadvantage**:
- The text shown in the editor has to be rendered in the preview as soon as the scrollbar is touched or new letters are written

**Comment by mwat56** [comment]
Ping ...

**Comment by linuxguy123** [comment]
I just came here to request the same functionality... synchronized scrolling.

I'd been looking for a good Markdown editor forever before finding Ghostwriter.   Ghostwriter rocks !  Best one I've found thus far.

**Comment by kdziuba-opgk** [comment]
+1 Looking forward to this feature.

**Comment by drmcnelson** [comment]
No, there really is a problem.

An example:  when I combined two paragraphs, by removing the space between them, the live display goes to the top of the document.

To bring it back in sync with the editor,  I have to delete a character within a paragraph and retype it, or manually drag the scroll bar in the preview.

So, this is broken.    Please fix it.

**Reply by linuxguy123** [reply]
I have the same problem.  Ghostwriter often gets lost and returns to the top of the document.   Really frustrating.

**Comment by ErlingSigurdson** [comment]
Please fix it, it's really broken. Synchronized scrolling is a must have option.
